src/components/testCaseLanding/qdm/TestCaseList.tsx

```diff
@@ -529,7 +538,7 @@ const TestCaseList = (props: TestCaseListProps) => {
                 onDeleteAllTestCases={() =>
                   setOpenDeleteAllTestCasesDialog(true)
                 }
-                onExportQRDA={() => exportQRDA()}
+                onExportQRDA={checkTestCases}
```

Comment:
I think the primary action here is to export QRDA with some validations. Maybe calling this function from `exportQRDA` would make it more clear.